### DISCUSSION TITLE
async

### DISCUSSION THREAD

**Discussion by christina-de-martinez** [christina-de-martinez/babel-plugin-glowup-vibes#38] [Polls]
Choose your favorite

**Comment by Cleman25** [comment]
Chill is more suitable for setTimeout

**Comment by dopenguin** [comment]
Would `letItCook` be a good fit for `async`?

**Reply by Jpobi** [reply]
This

**Reply by arienshibani** [reply]
I agree with @dopenguin. While `finna` is a colloquial expression that could fit, i believe that `letItCook` is a more descriptive and contextually relevant term.

**Reply by stifskere** [reply]
This is good

**Comment by Fern-Ali** [comment]
I like “peepThis” for async, and “realShit” for await 😂 I absolutely adore this idea and I am planning to contribute 🙌 you’re a goddess

**Reply by stifskere** [reply]
Maybe `realShit` is not clear enough for await, it doesn't actually have any related meaning.

**Comment by christina-de-martinez** [comment]
Accidentally pushed straight to main 🫣 but I went with holdUp and letItCook for async and await.
